app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  validates :author, presence: true
```

Comment:
- [ ] **[REQUIRED]** The comment model `belongs_to :user`  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/comment_belongs_to_a_user.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))

- [ ] **[REQUIRED]** Presence of `post` is being checked two times. Usually `belongs_to` would be enough in this situation. When I try to save an invalid comment using rails console, I get this:
```ruby
irb(main):023:0> c1 = Comment.new
irb(main):024:0> c1.save
=> false
irb(main):025:0> c1.errors.full_messages
=> ["Post must exist", "Post can't be blank", "Author can't be blank", "Body can't be blank"]
```
Notice, that it's checking twice for post. Please fix it.